src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private Ball ball = new Ball();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You should create a single instance of `ColorSupplier` in the class and use it in the `getRandomBall` method.